metadata-integration/java/datahub-client/src/main/java/datahub/client/rest/RestEmitter.java

```diff
@@ -154,7 +154,7 @@ public static RestEmitter createWithDefaults() {
   @Override
   public Future<MetadataWriteResponse> emit(MetadataChangeProposalWrapper mcpw,
       Callback callback) throws IOException {
-    return emit(this.eventFormatter.convert(mcpw), callback);
```

Comment:
why are we abandoning the previous member variable `eventFormatter` (class `EventFormatter` and replacing with a hard-coded `StringEscapeUtils.convert` static method? Can we instead improve the existing EventFormatter implementation to add this functionality? Then the unit test can be moved to the existing `EventFormatterTest` class as well.

Reply:
Missed this, yeah it makes sense to continue routing through the eventFormatter and to just modify the convert method rather than pass it directly to the util.